### PR TITLE
Post page header updates

### PR DESCRIPTION
**Pull request by s-cheng**
This PR is part of a broader project to update the design of the post page header on the EA Forum. Most of the changes should only affect the EA Forum, but one piece I kept in for LW since I thought it was an overall improvement. :) [Project figma here.](https://www.figma.com/file/d09iVzNSpZp9GyWGcfkWmX/Q2.2-2023?type=design&node-id=13-10576&t=N9AcuSwkuoGxzq4J-0)

In this PR, I've just done the bits that reorganize existing parts of the post page header. The major things are:
1. On wide screens, karma/voting has moved to the left of the title, and the title & secondary info take up the full col width. This makes the layout more consistent with post items.
2. On wide screens, the secondary info is split into two sections, which makes it easier to parse.
3. If the secondary info row would wrap, we split it at the end of the authors list, so the rest of that info is on a separate line. (<- **this is the only one that I didn't forum-gate**)
4. There's now a border between the post header and the tags, to help make the right-side icons not feel like they are floating.

Screen Shot 2023-06-15 at 1 58 23 AM
Screen Shot 2023-06-15 at 2 01 49 AM
Screen Shot 2023-06-15 at 2 16 01 PM

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1204837482038052) by [Unito](https://www.unito.io)

### CODE REVIEW

File: packages/lesswrong/components/posts/PostsPage/PostsPageDate.tsx [hunk 1]
```diff
@@ -50,13 +50,11 @@ const PostsPageDate = ({ post, hasMajorRevision, classes }: {
...
-        <span className={classes.date}>
-          <FormatDate date={post.postedAt} format={format} tooltip={false} />
-        </span>
-    </LWTooltip>
-  </React.Fragment>);
+  return <LWTooltip title={tooltip} placement="bottom">
+    <span className={classes.date}>
+      <FormatDate date={post.postedAt} format={format} tooltip={false} />
+    </span>
+  </LWTooltip>
```
Comment (s-cheng) [inline packages/lesswrong/components/posts/PostsPage/PostsPageDate.tsx:57]: This is just code cleanup, no functionality change

**Comment by darkruby501** [comment]
I like this change! Thanks

File: packages/lesswrong/components/posts/PostsPage/PostsPagePostHeader.tsx [hunk 2]
```diff
@@ -271,6 +293,57 @@ const PostsPagePostHeader = ({post, answers = [], dialogueResponses = [], toggle
...
+  
+  const tripleDotMenuNode = !hideMenu &&
+    <span className={classes.actions}>
+      <AnalyticsContext pageElementContext="tripleDotMenu">
+        <PostActionsButton post={post} />
+      </AnalyticsContext>
+    </span>
+  
+  // this is the info section under the post title, to the right of the author names
+  let secondaryInfoNode = <div className={classes.secondaryInfo}>
```
Comment (jpaddison3) [inline packages/lesswrong/components/posts/PostsPage/PostsPagePostHeader.tsx:312]: This section of this file is really pleasant to read. Nice work

**Review by jpaddison3 — APPROVED** [review]
LGTM

**Comment by s-cheng** [comment]
@agnesstenlund Thanks so much! ^^ I've fixed the action icon color, and I also moved the post page voting tooltips to the left of the arrows so it doesn't overlap the post title.

**Pull request merged**